extension/apmconfigextension/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+type: apmconfig
+
+status:
+  class: extension
+  stability:
+    development: [extension]
+  distributions: []
+  codeowners:
+    active: [rogercoll]
```

Comment:
Should a codeowner always be a person or a team?

Reply:
Good point, do we have a team's handle? I think using `otel-dev` involves many subteams.

Reply:
@sboomsma can possibly set up a new team for the (new) otel data team. Would that be sufficient?